profile-widget/profilescene.cpp

```diff
@@ -29,19 +42,63 @@ const static struct ProfileItemPos {
 	ProfileItemPos();
 } itemPos;
 
+template<typename T, class... Args>
```

Comment:
another commit where I wish for a better review UI... I think this is the same code just moved around - but that's so painful to verify...